cli/coda.ts

```diff
@@ -83,6 +84,18 @@ if (require.main === module) {
       },
       handler: handleRegister,
     })
+    .command({
+      command: 'whoami [apiToken]',
```

Comment:
seems like this should be `whoami <apiToken>` since it's optional.

Reply:
I think `[ ]` does indicate optional args.

Reply:
oh my mistake.

Reply:
I essentially copied the `register` command for that format. Seems like it's not super intuitive to a user at a glance how `<requiredArgument>` and `[optionalArgument]` in the yargs library work